### DISCUSSION TITLE
Filter by number/count of relations, is it possible?

### DISCUSSION THREAD

**Discussion by MorenoMdz** [prisma/prisma#21815] [Q&A] [answered]
### Question

For example, get all orders where the number of items equal 1.

```ts
this.prisma.order.findMany({
      where: {
        items: {
          _count: {
            equals: 1,
          },
        },
      },
      include: {
        items: true, 
      },
    })
  }
  ```
  
  Is it possible?

**Comment by ludralph** [comment, marked as answer]
Hi @MorenoMdz 👋 

Thank you for raising this question.

This is not possible at the moment. We do have an open [feature request](https://github.com/prisma/prisma/issues/8935) to allow searching by relation count. Do add a 👍  to the issue to enable our engineers prioritize it. You can also add your use case to the issue.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
